server/src/main/java/org/opensearch/monitor/os/OsProbe.java

```diff
@@ -440,7 +440,7 @@ String readSysFsCgroupCpuAcctCpuAcctCfsQuota(final String controlGroup) throws I
     @SuppressForbidden(reason = "access /sys/fs/cgroup/cpu")
     List<String> readSysFsCgroupCpuAcctCpuStat(final String controlGroup) throws IOException {
         final List<String> lines = Files.readAllLines(PathUtils.get("/sys/fs/cgroup/cpu", controlGroup, "cpu.stat"));
-        assert lines != null && lines.size() == 3;
+        assert lines != null && lines.isEmpty() == false;
```

Comment:
Just thinking out loud. In `getCgroupCpuAcctCpuStat` we will assert if any of the expected stat values is missing. Is it better to assert that `.size() >= 3` here or to get an explicit assert that, for example, `nr_periods` is missing? Probably the latter (code as is), but wanted to ask.

Reply:
Yes you are right better to check with `>=3`. The reason I did not add it only due to a magic number :-). And check for addition/new properties which could appear in the future IMHO better to implement via logging. What do you think? I can create a feature request and implement it for additional fields and logging.

Reply:
1 sec will change to `>= 3` in 1 sec

Reply:
I would leave this as `== false` and add an assert into `getCgroupCpuAcctCpuStat` that says "expected to see these values, but saw those values" instead.

Reply:
Ok. As a separate PR or in this one?

Reply:
Done